src/stimulus/data/encoding/encoders.py

```diff
@@ -88,13 +98,6 @@ class TextOneHotEncoder(AbstractEncoder):
     Attributes:
         alphabet (str): the alphabet to one hot encode the data with
         encoder (OneHotEncoder): preprocessing.OneHotEncoder object initialized with self.alphabet
-
```

Comment:
Why is tthis deleted ?

Reply:
because the documentation already builds method reference even without the method chunk in the docstring, otherwise i see two time the method chunk in the wiki.

Reply:
probably this should be a tweak to the mkdocs config rather than a change in the documentation here, we can merge this PR but I will look into it